app/src/main/java/org/gdg/frisbee/android/adapter/ChapterAdapter.java

```diff
@@ -44,21 +39,27 @@ public ChapterAdapter(Context context, int textViewResourceId) {
         mInflater = LayoutInflater.from(context);
     }
 
-    public ChapterAdapter(Context context, int textViewResourceId, List<Chapter> objects) {
-        super(context, textViewResourceId, objects);
-        mInflater = LayoutInflater.from(context);
+    @Override
+    public View getView(final int position, final View convertView, final ViewGroup parent) {
+        View view = convertView;
+        if (view == null) {
+            view = mInflater.inflate(R.layout.spinner_item_actionbar, parent, false);
```

Comment:
The constructor does not need the "textViewResourceId" (which was a layout resource anyway)